src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -178,23 +180,37 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
         }
 
         diff.setBuildFinished(build.getResult());
+        String customComment = null;
+        try {
+            customComment = getRemoteComment(build, logger, this.commentFile);
+        } catch(InterruptedException e) {
+            e.printStackTrace(logger);
+        } catch (IOException e) {
+            Util.displayIOException(e, listener);
+
+        }
+        if (customComment != null) {
```

Comment:
Can you just move this logic inside the try? No point in executing it if we hit one of the exceptions. Also, you can use CommonUtils.isBlank() to check for null or empty strings